backend/server.go

```diff
@@ -1,27 +1,26 @@
 package backend
 
 import (
-    "fmt"
```

Comment:
Let's revert these formatting changes (for now). Since i can't tell what else has changed. I just ran go fmt from my branch so I'm confused to see any changes at all.